src/gluonts/dataset/pandas.py

```diff
@@ -62,13 +73,21 @@ class PandasDataset:
         For target and past dynamic features last ``ignore_last_n_targets``
         elements are removed when iterating over the data set. This becomes
         important when the predictor is called.
+    unchecked
+        Whether consistency checks on indexes should be skipped.
+        (Default: ``False``)
+    assume_sorted
+        Whether to assume that indexes are sorted by time, and skip sorting.
+        (Default: ``False``)
     """
 
     dataframes: Union[
         pd.DataFrame,
         pd.Series,
-        List[pd.DataFrame],
```

Comment:
I think we can make use of TypeVar here: https://stackoverflow.com/questions/59933946/difference-between-typevart-a-b-and-typevart-bound-uniona-b